omnibus.rb

```diff
@@ -44,9 +44,14 @@
 fetcher_retries 3
 fetcher_read_timeout 120
 
-# We limit this to 10 workers to eliminate transient timing issues in the
-# way Ruby (and other components) compiles on some more esoteric *nixes.
-workers 10
+# Work around for solaris zones, provide a less-bad number of workers
+if solaris? && Ohai['cpu']['total'] == 0
+  workers 10
+end
```

Comment:
That seems like it should be an 'unless'?  The comment also isn't clear as to if 'less bad' is supposed to be smaller or larger which might be confusing me as well...  I'm guessing the problem here is that 10 is too many on our solaris zones.  I'm also not sure that this isn't specific to how we happen to have our zones setup and if this wouldn't be better put in the code which renders the omnibus.rb on our internal build slaves...  Also a little surprised that we're not overwriting this file...